### PR TITLE
add apt-get update to install-test-deps script

### PR DESCRIPTION
**Pull request by meandavejustice**

### CODE REVIEW

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3814/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/ad874937d95755f45ceda2aa3e8bbbda5dbeda09/

**Comment by b4handjr** [comment]
Can we re-enable the tests in the circle.yml?

**Comment by fzzzy** [comment]
Yes, let's re-enable the integration tests in this pr before merging it.

**Comment by fzzzy** [comment]
While we are at it, let's also disable the intermittent newsletter form test so that hopefully the integration tests pass more reliably going forward.

**Review by fzzzy — CHANGES_REQUESTED** [review]
Please re-enable the integration tests and disable the intermittent newsletter form test in this branch.

**Comment by b4handjr** [comment]
> While we are at it, let's also disable the intermittent newsletter form test so that hopefully the integration tests pass more reliably going forward.

Which test is that exactly?

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3814/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/3c01c8874ee95813138b5266d23a79d834df1664/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3814/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/2b36f68c7cfc6f4cea575ed59ddda25149fa2693/

File: frontend/test/ui/test_homepage.py [hunk 1]
```diff
@@ -65,13 +65,13 @@ def test_support_pages(base_url, selenium, page, title):
...
 
 
-@pytest.mark.nondestructive
-def test_incomplete_email(base_url, selenium):
-    """Test signup with incorrect email"""
-    page = Home(selenium, base_url).open()
-    assert page.signup_footer.is_stay_informed_displayed
-    page.signup_footer.sign_up('test')
-    assert page.signup_footer.is_stay_informed_displayed
+# @pytest.mark.nondestructive
```
Comment (b4handjr) [inline frontend/test/ui/test_homepage.py]: Just use this: https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions

**Review by b4handjr — CHANGES_REQUESTED** [review]

File: frontend/test/ui/test_homepage.py (hunk 1 above)
Comment (b4handjr) [reply frontend/test/ui/test_homepage.py]: A reason why other than "Its flaky" would be good too as I'd like to fix it.

File: circle.yml [hunk 2]
```diff
@@ -153,9 +153,24 @@ workflows:
...
           requires:
             - build
+      - integration_test:
+          requires:
+            - build
+          filters:
+            branches:
+              ignore:
+                - l10n
+      - storybook_deploy:
```
Comment (fzzzy) [inline circle.yml]: We have to disable the storybook deploy for now until we have cycles to work on it.

**Review by fzzzy — CHANGES_REQUESTED** [review]

File: frontend/test/ui/test_homepage.py (hunk 1 above)
Comment (fzzzy) [reply frontend/test/ui/test_homepage.py]: @jrbenny35 you can look through the circle history to see it failing intermittently. it is unable to find the element on the page even though it is there.

File: frontend/test/ui/test_homepage.py [hunk 3]
```diff
@@ -65,6 +65,8 @@ def test_support_pages(base_url, selenium, page, title):
     assert title in page.title
 
 
+@pytest.mark.skip(reason="Test is really inconsistent, hard to
```
Comment (b4handjr) [inline frontend/test/ui/test_homepage.py]: I think you need to do a ```\``` so the string extends to the next line.

**Review by b4handjr — CHANGES_REQUESTED** [review]

**Review by b4handjr — APPROVED** [review]

**Review by fzzzy — APPROVED** [review]

**Pull request merged**